### PR TITLE
Wednesday afternoon pull

### PR DESCRIPTION
**Pull request by DandelionSprout**
Also aims to solve #583.

### CODE REVIEW

**Comment by collinbarrett** [comment]
Thanks! disregard that travis-ci failure, it's unrelated to your PR. I'll review and merge.

File: data/FilterList.json [hunk 1]
```diff
@@ -965,7 +965,9 @@
     "publishedDate": "2018-05-23T21:53:55",
     "syntaxId": 4,
     "updatedDate": "2018-10-09T01:28:00",
-    "viewUrl": "https://raw.githubusercontent.com/NanoAdblocker/NanoFilters/master/NanoMirror/NanoDefender.txt"
+    "viewUrl": "https://raw.githubusercontent.com/NanoAdblocker/NanoFilters/master/NanoMirror/NanoDefender.txt",
+    "viewUrlMirror1": "https://raw.githubusercontent.com/NanoAdblocker/NanoFilters/master/NanoMirror/NanoDefender.txt",
```
Comment (collinbarrett) [inline data/FilterList.json]:
```suggestion
    "viewUrlMirror1": "https://cdn.rawgit.com/NanoAdblocker/NanoFilters/master/NanoMirror/NanoDefender.txt",
```

File: data/FilterList.json [hunk 2]
```diff
@@ -965,7 +965,9 @@
...
+    "viewUrlMirror2": "https://cdn.rawgit.com/NanoAdblocker/NanoFilters/master/NanoMirror/NanoDefender.txt"
```
Comment (collinbarrett) [inline data/FilterList.json]:
```suggestion
```

Comment (collinbarrett) [reply data/FilterList.json]: Not sure if the beta GitHub requested change feature works properly. But, essentially, let's not duplicate the same url in both viewUrl and viewUrlMirror1.

**Comment by DandelionSprout** [comment]
Well spotted. It should hopefully have been resolved now. And yes, I don't think the "Suggested change" function is working any well at all.

**Pull request merged**